src/ios/GeolocationPlugin.m

```diff
@@ -0,0 +1,276 @@
+//
+//  GeolocationPlugin.m
+//  Geolocation
+//
+//  Created by dev on 2/9/16.
+//
+//
+
+#import "GeolocationPlugin.h"
+
+@implementation GeolocationPlugin
+
+-(void)start:(CDVInvokedUrlCommand *)command
+{
+    coord1 = [command.arguments objectAtIndex:0];   // Get center and radio of region1
+    coord2 = [command.arguments objectAtIndex:1];   // Get center and radio of region2
+    url = [command.arguments objectAtIndex:2];      // Get server url
+
```

Comment:
glad you changed the method names to lower case. we can add some more improvement in your coding style:

* at most one empty line between code blocks within methods
* two empty lines between methods

I'll add some more below. This are minor in terms of functionality, but really improve the code readability and quality.